src/DDTrace/version.php

```diff
@@ -1,5 +1,13 @@
 <?php
 
-// Must begin with a number for Debian packaging requirements
-// Must use single-quotes for packaging script to work
-return '1.0.0-nightly'; // Update Tracer::VERSION too
+/**
+ * Returns the tracer sources versions.
+ *
+ * @return string
+ */
+function ddtrace_src_version()
+{
+    // Must begin with a number for Debian packaging requirements
+    // Must use single-quotes for packaging script to work
+    return '1.0.0-nightly'; // Update Tracer::VERSION too
+}
```

Comment:
I like the idea of adding this to the generated file!

I'm with Levi on making the version a constant in the `DDTrace` namespace. And we can still `return` the version number from the file so that we don't have to change the build tools/dd-doctor. :)

Reply:
> 'm with Levi on making the version a constant in the DDTrace namespace

Like it!

> And we can still return the version number from the file so that we don't have to change the build tools/dd-doctor. :)

I think this is not possible if we add it to generated file, as a `return` statement would cause the script to, well, return :). We could put it as the last imported file, but to be honest I'd prefer to remove one potential source of troubles given that the price to pay is limited. What do you think?

Reply:
Ah, good point. :) Sounds good to me! 👍